mkinitcpio

```diff
@@ -211,6 +211,9 @@ build_image() {
         lz4)
             COMPRESSION_OPTIONS+=('-l')
             ;;
+        zstd)
+            COMPRESSION_OPTIONS+=('-22' '--ultra')
```

Comment:
the options used are the same as found in the kernel proposal for the vmlinux image

Reply:
You have a link for this?

Reply:
https://lkml.org/lkml/2020/7/23/1013
But I've actually read the documentation and -22 shouldn't be used, instead for initramfs -19 should do the job